examples/grpc-kotlin/src/main/kotlin/example/armeria/grpc/kotlin/HelloExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package example.armeria.grpc.kotlin
+
+import example.armeria.grpc.kotlin.Hello.HelloReply
+import example.armeria.grpc.kotlin.Hello.HelloRequest
+
+fun String.buildHelloReply(): HelloReply = HelloReply.newBuilder().setMessage(this).build()
+
+fun String.buildHelloRequest(): HelloRequest = HelloRequest.newBuilder().setName(this).build()
+
+fun String.hello(): String = "Hello, $this!"
```

Comment:
This is not related to the Armeria core feature. However, I think string extension methods are not a good example to build a Message because `String` is a common class that represents char sequences. It looks fancy but I feel it is a bit abused.

How about remove the string extension methods and add a class like `HelloMessageExtensions` or `HelloMessageFatory`?

```kt
class HeloMessageFatory {
  fun newHelloReply(message: String): HelloReply = ...
  fun newHelloRequest(message: String): HelloRequest = ...
  ...
}
```

Reply:
I agree. Usually in Kotlin, adding `Extension` to the end of a class name has a different purpose, so we'll change it to `HelloMessageFatory`.